src/mapml/layers/FeatureLayer.js

```diff
@@ -13,17 +13,23 @@ export var MapMLFeatures = L.FeatureGroup.extend({
         // info: https://github.com/Leaflet/Leaflet/pull/4597
         L.DomUtil.addClass(this._container, 'leaflet-pane mapml-vector-container');
         L.setOptions(this.options.renderer, {pane: this._container});
-        let anim = L.DomUtil.create("style", "mapml-feature-animation", this._container);
-        anim.innerHTML = `@keyframes pathSelect {
-          0% {stroke: white;}
-          50% {stroke: black;}
-        }
-        g:focus > path,
-        path:focus {
-          animation-name: pathSelect;
-          animation-duration: 1s;
-          stroke-width: 5;
-          stroke: black;
+        let style = L.DomUtil.create("style", "mapml-feature-style", this._container);
+        style.innerHTML = `
+        .mapml-vector-container g[role="link"]:focus,
+        .mapml-vector-container g[role="link"]:hover,
+        .mapml-vector-container g[role="button"]:focus,
+        .mapml-vector-container g[role="button"]:hover,
+        .mapml-vector-container g[role="link"] path:focus,
+        .mapml-vector-container g[role="link"] path:hover,
+        .mapml-vector-container g[role="button"] path:focus,
+        .mapml-vector-container g[role="button"] path:hover,
+        .mapml-vector-container g[role="link"]:focus path,
+        .mapml-vector-container g[role="link"]:hover path,
+        .mapml-vector-container g[role="button"]:focus path,
+        .mapml-vector-container g[role="button"]:hover path {
```

Comment:
Scoping these styles to `.mapml-vector-container` may be problematic, for example they wouldn't apply to the SVG `path`/`g` elements in https://maps4html.org/experiments/vector-tiles/mapml/mapml-viewer/ because that SVG is created inside `.mapml-templatedlayer-container`.

I'll remove that scope, perhaps it's better to scope the styles `.leaflet-pane` (alternatively `.mapml-layer`?), otherwise these styles may apply inside things like `.leaflet-controls-container` which we probably don't want.

Reply:
> they wouldn't apply to the SVG `path`/`g` elements in https://maps4html.org/experiments/vector-tiles/mapml/mapml-viewer/

While those `path`/`g` elements don't seem to be interactive, I suppose they could be. So it's safer to remove the `.mapml-vector-container` scope for now. Done in https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/478/commits/bc604f014e24ac69b8f9e04e8da947a695daf2e3.